pkgs/by-name/to/tone/package.nix

```diff
@@ -22,12 +29,18 @@ buildDotnetModule rec {
   dotnet-sdk = dotnetCorePackages.sdk_6_0;
```

Comment:
Also set `dotnet-runtime`?

Reply:
Seems like that should be consistent. Done.